.travis.yml

```diff
@@ -1,6 +1,16 @@
 language: android
+sudo: true # travis-ci/travis-ci#5582 implies this could help against killed build-jobs
 before_script: ./pre-build.sh && sed -i "s#com.android.tools.build:gradle:.*'#com.android.tools.build:gradle:1.5.0'#" build.gradle
-jdk: openjdk7
+jdk:
+    - openjdk7
+    - oraclejdk7
+    - oraclejdk8
```

Comment:
Since this application is Free Software, I would prefer not building it with non-free software here. What is the advantage of adding the proprietary JDK here?